.github/workflows/workflow.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+FILE_PATH="$1"
+
+# Arguments other than file path are passed to the command's arguments
+shift 1
+
+if [ ! -f "$FILE_PATH" ]; then
```

Comment:
```suggestion
if [ ! -f "$FILE_PATH" ]
then
```

Reply:
fixed in https://github.com/tier4/scenario_simulator_v2/pull/1223/commits/bbcc4a618d1651a303931d0592e9026937614a0f